CSharp/BatchExplorer/ViewModels/MainViewModel.cs

```diff
@@ -733,6 +733,32 @@ public CommandBase RefreshJobs
         }
 
         /// <summary>
+        /// Delete selected jobs
+        /// </summary>
+        public CommandBase DeleteSelectedJobs
+        {
+            get
+            {
+                return new CommandBase(
+                    (o) =>
+                    {
+                        foreach (var job in this.jobs)

```

Comment:
Since this could be a very long operation (imagine you have 100's or 1000's of jobs) I would consider turning on the spinner on the left pane while you're executing the foreach loop